component/grant-type/src/main/java/org/wso2/carbon/identity/oauth2/grant/jwt/JWTBearerGrantHandler.java

```diff
@@ -866,6 +862,32 @@ private boolean validateSignature(SignedJWT signedJWT, IdentityProvider idp)
         }
     }
 
+    /**
+     * Check the validity of the x509Certificate.
+     *
+     * @param x509Certificate   x509Certificate
+     * @throws IdentityOAuth2Exception
+     */
+    private void checkValidity(X509Certificate x509Certificate) throws IdentityOAuth2Exception {
+
+        String isEnforceCertificateValidity = IdentityUtil.getProperty(ENFORCE_CERTIFICATE_VALIDITY);
+        if (StringUtils.isNotEmpty(isEnforceCertificateValidity)
+                && !Boolean.parseBoolean(isEnforceCertificateValidity)) {
```

Comment:
Boolean.parseBoolean will internally do a null check and return false. Therefore StringUtils.isNotEmpty is redundant here

Reply:
If this property is not configured the isEnforceCertificateValidity will be null (or empty), In this case, we need to enforce the validation (The default behaviour). So StringUtils.isNotEmpty  is not redundant.